front/src/components/App/SalesArea/SalesArea.tsx

```diff
@@ -1,17 +1,38 @@
 import Button from "../../UI/Button/Button";
 import RouteLink from "../../UI/RouteLink/RouteLink.styled";
 import { APP_PAGE } from "../App";
+import allegro_image from "../../Images/allegro.jpg";
+import olx_image from "../../Images/olx.jpg";
+import ebay_image from "../../Images/ebay.jpg";
+import CompaniesSector from "../../UI/CompanyView/CompaniesSector";
 import { SalesAreaStyled } from "./SalesArea.styled";
 
 const SalesArea = () => {
+  const examples = [
```

Comment:
jeśli definujemy taką stała to nie umieszczamy jej w komponencie, ponieważ nie chcemy żeby przy każdym przerenderowaniu komponentu na nowo musiała się tworzyć cała lista. Zamiast tego możesz to wynieść po prostu ponad definicję SalesArea.

Druga uwaga to nazewnictwo. Samo examples nie jest złe, ale nie mówi od razu co tam się znajduje. Zamiast tego można to nazwać np. EXAMPLE_COMPANIES (wielkimi literami żeby zaznaczyć, że nigdzie nie powinno to być modyfikowane).